tests/e2e_test.go

```diff
@@ -0,0 +1,111 @@
+package test
+
+import (
+	"context"
+	"encoding/json"
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+	"strings"
+	"testing"
+
+	"github.com/sigstore/cosign/pkg/cosign"
+	"github.com/sigstore/root-signing/cmd/tuf/app"
+	vapp "github.com/sigstore/root-signing/cmd/verify/app"
+
+	"github.com/theupdateframework/go-tuf"
+	"github.com/theupdateframework/go-tuf/data"
+)
+
+// Create test HSM keys.
```

Comment:
nit: This is a single key, correct?

I wonder if it's worth creating multiple keys to mirror production

Reply:
It is just a single key.
 I mean to add a TODO -- Init doesn't require signing with anything, and there's only one test_data key in there. 

Adding a TODO to create keys programmatically, and also to test >1 threshold.